modelforge/potential/utils.py

```diff
@@ -558,7 +560,7 @@ def forward(self, input: torch.Tensor):
 
         """
         y = F.linear(input, self.weight, self.bias)
-        return self.activation_function(y)
+        return self.activation_function_class(y)
```

Comment:
This might be difficult to understand, and I believe it is inconsistent with `nn.Identity()`.

We provide the class name, but then we initialize an instance and use it in the layers. We can also use the callable version since activity functions don't have an internal state. But here `nn.Identity()` is an instance, but the variable name implies it is a class.

Reply:
I think this is the better solution! Thanks!